tests/commands/test_store_api.py

```diff
@@ -656,8 +656,8 @@ def test_upload_straightforward(client_mock, emitter, config):
     with patch.dict("charmcraft.commands.store.store.UPLOAD_ENDING_STATUSES", fake_statuses):
         result = store._upload(test_endpoint, test_filepath)
 
-    # check all client calls
-    assert client_mock.mock_calls == [
+    # check all client calls, excpet whoami() in first 10 calls
```

Comment:
```suggestion
    # check all client calls, expect whoami() in first 10 calls
```

(please fix the other `expect` typos in this file too)